weave

```diff
@@ -1667,7 +1667,14 @@ launch_router() {
            --label=weavevolumes $WEAVEDB_IMAGE >/dev/null
     fi
 
-    RESOLV_CONF=$(chroot ${HOST_ROOT:-/} readlink -f /etc/resolv.conf)
+    if [ -f ${HOST_ROOT:-/}/etc/resolv.conf ]; then # regular file
```

Comment:
Sadly `-f` which tests for "regular file" in this context does not exclude symbolic links to files; we need a test such as `-L`